src/add-storybook/utils/constants.ts

```diff
@@ -0,0 +1,19 @@
+export const dependencies = {
+    "@babel/core": "^7.17.9",
+    "@compodoc/compodoc": "^1.1.19",
+    "@storybook/addon-actions": "^6.4.20",
+    "@storybook/addon-essentials": "^6.4.20",
+    "@storybook/addon-interactions": "^6.4.20",
+    "@storybook/addon-links": "^6.4.20",
+    "@storybook/angular": "^6.4.20",
+    "@storybook/builder-webpack5": "^6.4.20",
+    "@storybook/manager-webpack5": "^6.4.20",
```

Comment:
maybe you would like to use a constant for the version.

Reply:
solved